### DISCUSSION TITLE
How to replace Variable in Bruno

### DISCUSSION THREAD

**Discussion by HussmohaCode** [usebruno/bruno#1718] [Q&A]
My Code in the Script Tab. Below is my code for replacing the Variable
**bru.setEnv("Username",bru.variables.replaceIn)('{{$randomCatchPhrase}}'+'{{$randomAbbreviation}}'+'{{$randomInt}}'));**
Every time I get undefined for "variables" and "replaceIn" words. What would be the correct code for Variable replacement in the Environment? 
Thank you

**Comment by Theqwertypusher** [comment]
Hey there, seems like you might be using a JS API found in postman which isn't available in Bruno. The [Bruno docs](https://docs.usebruno.com/scripting/javascript-reference.html) lists all the available API you can use in the script tab.

To your question, you can use bru.setEnvVar( [ENV_KEY], [value] )

example: 
```function onResponse(res) {
let data = res.getBody();
let token = bru.setEnvVar("access_token", data.token);
}
```
